tests/io_perfomance/image-volume/README.md

```diff
@@ -0,0 +1,46 @@
+# FIO autotests container
+
+This container runs 36 FIO tests of different types and loads in turn. You can find out more about the test configuration by looking at the config.fio file in the container itself, or the test results in the specified GitHub repository in the directory. Each test is 2 minutes long. The total approximate testing time is ~ 1 hour 12 minutes. Run it and just wait for the result in the specified repository.
+
+## How to
+
+Before starting the container, you need to [get a token on GitHub.com](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token).
+
+### To deploy
+
+```console
+./eden pod deploy --metadata="EVE_VERSION=5.14\nGIT_REPO=eve-performance\nGIT_LOGIN=itmo-eve\nGIT_TOKEN=1111111111111111111111111aaaaaaaaaaaaaaa" -p 8029:80 docker://itmoeve/fio_tests --no-hyper
```

Comment:
As @mydatascience pointed out -- you can discover EVE_VERSION setting.

In fact, the approach I'd recommend @vk-en is to discover/default as much as you can so that I can run this test with only GIT_TOKEN specified and everything else being auto-discovered/defaulted.

Reply:
I will change the parameter for EVE_VERSION in the example.

I would not want to limit this test to one account and a repository, my solution allows you to use any repository and an account with access to it as a target for saving the results. I believe that this will allow this test to pass not only for our employees who have access to the itmo repository

Reply:
We can't discover repo without GIT_LOGIN / GIT_REPO